core/server/common/src/main/java/alluxio/underfs/AbstractUfsManager.java

```diff
@@ -185,6 +188,17 @@ private UnderFileSystem getOrAddWithRecorder(AlluxioURI ufsUri,
    */
   protected abstract void connectUfs(UnderFileSystem fs) throws IOException;
 
+  /**
+   * To check whether the filesystem is available by calling exists.
+   *
+   * @param fs the filesystem
+   * @param uriPath the UFS path
+   * @throws Exception
+   */
+  private void tryUseFileSystem(UnderFileSystem fs, String uriPath) throws Exception {
```

Comment:
As the `uriPath` represents the UFS path, how about renaming it to `ufsUriPath` or `ufsPath`?